managed/models/database.go

```diff
@@ -1161,6 +1257,11 @@ func migrateDB(db *reform.DB, params SetupDBParams) error {
 		}
 
 		if params.SetupFixtures == SkipFixtures {
+			err := EncryptDB(tx, params, itemsToEncrypt)
+			if err != nil {
+				return err
+			}
+
```

Comment:
can we move it up before `if`, so we don't have to have it both here and below

Reply:
Nice catch, moved.